webpack.config.js

```diff
@@ -0,0 +1,78 @@
+const webpack = require('webpack');
+const path = require('path');
+
+// Config directories
+const SRC_DIR = path.resolve(__dirname, 'src');
+const OUTPUT_DIR = path.resolve(__dirname, 'dist');
+const defaultInclude = [SRC_DIR];
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const { spawn } = require('child_process');
+
+module.exports = {
+  // original: entry: SRC_DIR + '/index.js'
+  entry: `${SRC_DIR}/main/index.js`,
+  output: {
+    path: OUTPUT_DIR,
+    publicPath: '/dist/',
+    filename: 'bundle.js',
+  },
+
+  module: {
+    rules: [
+      {
+        test: /\.css$/,
+        use: [{ loader: 'style-loader' }, { loader: 'css-loader' }],
+        include: defaultInclude,
+      },
+      {
+        test: /\.jsx?$/,
+        use: [{
+          loader: 'babel-loader',
+          options: {
+            presets: ['@babel/preset-env'],
+            plugins: ['@babel/plugin-proposal-object-rest-spread'],
+          },
+        }],
+        include: defaultInclude,
+      },
+      {
+        test: /\.(jpe?g|png|gif)$/,
+        use: [{ loader: 'file-loader?name=img/[name]__[hash:base64:5].[ext]' }],
+        include: defaultInclude,
+      },
+      {
+        test: /\.(eot|svg|ttf|woff|woff2)$/,
+        use: [{ loader: 'file-loader?name=font/[name]__[hash:base64:5].[ext]' }],
+        include: defaultInclude,
+      },
+    ],
+  },
+  target: 'electron-renderer',
+  devtool: 'cheap-source-map',
```

Comment:
[Webpack’s docs](https://webpack.js.org/configuration/devtool/) note that `cheap-source-map` is _not_ ideal for dev or build processes, but may be required for some third-party tools. Worth looking into if we notice slow build times later on.

Reply:
I agree, It was in the tutorial as well as in the webpack config for Swell.  if we discover we are not utilizing it, we can remove it.